jenkins/post_comment.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+
+ERRORS_FILENAME=$WORKSPACE/erros.log
+
+# This script is kicked with $1=0 
+# only if "bash build_and_test.sh" has exited successfully.
+if [[ $1 -eq 0 ]]; then
+  BODY="Test(${BUILD_DISPLAY_NAME}, ${GIT_COMMIT}) passed in $(uname -n)."
+else
+  BODY="Test(${BUILD_DISPLAY_NAME}, ${GIT_COMMIT}) failed in $(uname -n). 
+\`\`\`$(cat ${ERRORS_FILENAME})\`\`\`"
```

Comment:
It is possible to show not all messages but only a few results?
It obstructs review if the result is huge.

Reply:
Updated to display the first 50 lines (and mention there are more).